services/blockchain-indexer/shared/dataService/business/interoperability/states.js

```diff
@@ -0,0 +1,24 @@
+/*
+* LiskHQ/lisk-service
+* Copyright © 2022 Lisk Foundation
+*
+* See the LICENSE file at the top-level directory of this distribution
+* for licensing information.
+*
+* Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+* no part of this software, including this file, may be copied, modified,
+* propagated, or distributed except according to the terms contained in the
+* LICENSE file.
+*
+* Removal or modification of this copyright notice is prohibited.
+*
+*/
+const appstatus = {
+	ACTIVE: 'active',
+	REGISTERED: 'registered',
+	TERMINATED: 'terminated',
+};
+
+module.exports = {
+	appstatus,
```

Comment:
Please make the necessary corresponding changes. Also, please rename the file to `constants.js`.
```suggestion
const APP_STATE = {
	ACTIVE: 'active',
	REGISTERED: 'registered',
	TERMINATED: 'terminated',
};

module.exports = {
	APP_STATE,
```